### PR TITLE
feat: [ui]In the app selection list, add all installed applications f…

### PR DESCRIPTION
**Pull request by scx005548**
…or the user

In the app selection list, add all installed applications for the user.

Log: In the app selection list, add all installed applications for the user.

### CODE REVIEW

File: src/plugins/data-transfer/core/CMakeLists.txt [hunk 1]
```diff
@@ -56,7 +56,7 @@ else()
 endif()
 
 find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core Gui Network Svg)
```
Comment (re2zero) [inline src/plugins/data-transfer/core/CMakeLists.txt:58]: if (CMAKE_SYSTEM MATCHES "Windows")
find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Svg WinExtras)
find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Svg WinExtras)
endif()

File: src/plugins/data-transfer/core/CMakeLists.txt [hunk 2]
```diff
@@ -96,6 +96,7 @@ if (CMAKE_SYSTEM MATCHES "Windows")
         Qt5::Sql
```
Comment (re2zero) [inline src/plugins/data-transfer/core/CMakeLists.txt:96]: ${QUAZIP_LIB_TARGET_NAME}
        Qt${QT_VERSION_MAJOR}::Sql
        Qt${QT_VERSION_MAJOR}::Svg
        Qt${QT_VERSION_MAJOR}::WinExtras

File: src/plugins/data-transfer/core/CMakeLists.txt (hunk 1 above)
Comment (scx005548) [reply src/plugins/data-transfer/core/CMakeLists.txt]: done

File: src/plugins/data-transfer/core/CMakeLists.txt (hunk 2 above)
Comment (scx005548) [reply src/plugins/data-transfer/core/CMakeLists.txt]: done

**Review by re2zero — APPROVED** [review]
/approve

**Pull request merged**